WordPress/Classes/Models/Notifications/Notification.swift

```diff
@@ -69,12 +76,31 @@ class Notification: SPManagedObject
     ///
     private var cachedBodyBlockGroups: [NotificationBlockGroup]?
 
+    /// Array that contains the Cached Property Names
+    ///
+    private static let cachedAttributes = Set(arrayLiteral: "body", "header", "subject", "timestamp")
+
+
 
+    /// When needed, nukes cached attributes
+    ///
+    override func willChangeValueForKey(key: String) {
+        super.willChangeValueForKey(key)
+
+        guard managedObjectContext?.concurrencyType == .MainQueueConcurrencyType else {
```

Comment:
The cached attributes are expected to always be used only from the Main Thread (when populating TableView Cells). It's just an optimization, to avoid overwork.

I'll add a note to clear it up, thanks!